.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 
 python:
-  - "3.6"
```

Comment:
Why are we deprecating Python 3.6?

Reply:
According to the dependency requirements, the matplotlib version no longer supports 3.6.